src/trio/_threads.py

```diff
@@ -52,6 +54,79 @@ class _ParentTaskData(threading.local):
 _thread_counter = count()
 
 
+class _ActiveThreadCount:
+    count: int
+    event: Event
+
+    def __init__(self) -> None:
+        self.count = 0
+        self.event = Event()
```

Comment:
@CoolCat467 recommended NamedTuple to not have to have an `__init__`, though code in this repo generally uses attr to handle default values in "dataclasses". See e.g. `EpollWaiters` in `src/trio/_core_io_epoll.py`.